layouts/_default/single.html

```diff
@@ -1,8 +1,11 @@
+{{ partial "head.html" . }}
+{{ partial "head.html" . }}
```

Comment:
Two heads

Reply:
oh boy, how did i miss that double copy paste one. thanks for your methodical review.